src/content/certifications/programs/security-automation.json

```diff
@@ -21,6 +21,17 @@
 			},
 			"faqSlug": "vault-associate-002"
 		},
+		{
+			"title": "Vault Associate",
+			"examCode": "003",
+			"productSlug": "vault",
+			"versionTested": "Vault 1.16",
+			"description": "The Vault Associate certification is for Cloud Engineers specializing in security, development, or operations who know the basic concepts, skills, and use cases associated with Vault. You understand what Vault Enterprise features exist and can differentiate between Enterprise and Community Edition. You will be best prepared for this exam if you have professional experience using Vault in production, but performing the exam objectives in a personal demo environment may be sufficient.",
+			"links": {
+				"prepare": "/vault/tutorials/associate-cert-003"
+			},
+			"faqSlug": "vault-associate-003"
+		},
```

Comment:
It is weird that this PR adds this block even tho this block was already added in PR https://github.com/hashicorp/dev-portal/pull/2577. Do you need to rebase or are there going to be merge conflicts?

Reply:
I added this so you could see the preview b/c the auto merge means that I cannot rebase against main to pull that change in. So, it IS weird. :(

Reply:
Closed other PR and we will pull all needed info into this PR